### ISSUE TITLE
why the bloom-filters is invalid?

### ISSUE THREAD

**Issue by raymondluo1986** [prestodb/presto#14673]
Hi,team.

my presto server version is presto 0.213
my hive version is 2.3.6

###########################################################
in hive,I create table:
CREATE TABLE test_bloom(
  uuid_key STRING,
  media_file_name STRING,
  vehicle_id INT,
  play_auto_count INT,
) partitioned by (DataYear STRING, DataMonth STRING, DataDay STRING)
STORED AS ORC
TBLPROPERTIES('orc.create.index'='true','orc.bloom.filter.columns'='media_file_name');
############################################################
in presto,I set Catalog Properties:
connector.name=hive-hadoop2
hive.metastore.uri=thrift://192.168.1.100:9083
hive.orc.bloom-filters.enabled=true
############################################################
I execute SQL use presto-cli-0.213-executable.jar :

select
sum(play_auto_count) as play_auto_count,
sum(play_auto_duration) as play_auto_duration,
sum(play_vod_count) as play_vod_count,
sum(play_vod_duration) as play_vod_duration,
sum(play_all_duration) as play_all_duration,
sum(play_all_count) as play_all_count
from test_bloom
where
media_file_name='testMediaFile';
############################################################
in presto web ui show:

Input Rows	69.2K
Input Data	3.57MB
Raw Input Rows	30.1M
Raw Input Data	144.83MB
Peak User Memory	0B
Peak Total Memory	704B

############################################################
the data look like:
uuid_key  media_file_name  vehicle_id  play_auto_count
a1             testMediaFile         1             100
a2            testMediaFile          2             201
a3            testMediaFile         3             300
.
.
a8000     testMediaFile        8000          100
b1           testMediaFile2        1             100
b2           testMediaFile2        2             201
b3           testMediaFile2        3             300
.
.
b8000     testMediaFile2     8000          100
.
.
.

the Raw Input Rows is 30.1M,and the Input Rows is 69.2K。

Why the bloom-filters is invalid?

**Comment by mbasmanova** [comment]
@raymondluo1986 I'm not sure I understand the question. "Why the bloom-filters is invalid?" - What do you mean exactly? Would you explain?

**Comment by raymondluo1986** [comment]
@mbasmanova 
sorry!My English is the general as. 
in presto web ui show:

Input Rows 69.2K
Input Data 3.57MB
Raw Input Rows 30.1M
Raw Input Data 144.83MB
Peak User Memory 0B
Peak Total Memory 704B

the Raw Input Rows is 30.1M,and the Input Rows is 69.2K。

if the bloom-filters is run normally，the Raw Input Rows should be near 69.2K。

**Comment by mbasmanova** [comment]
@dilipkasana Is this something you'd know how to investigate?

**Comment by mbasmanova** [comment]
@raymondluo1986 This could be the same issue as https://github.com/prestodb/presto/issues/7120#issuecomment-478497728

**Comment by dilipkasana** [comment]
@raymondluo1986 can you post the metadata output for your orc file by executing following command :
1. hive --orcfiledump filename
2. java -jar orc-tools-1.6.3-uber.jar meta filename (https://orc.apache.org/docs/java-tools.html)

**Comment by raymondluo1986** [comment]
@mbasmanova @dilipkasana
My issue is the same issue as #7120 and #12900.
in order to fix my system exception, I executed the following step.
(1) I modified the statement to create the table.
CREATE TABLE test_bloom(
uuid_key STRING,
media_file_name STRING,
vehicle_id INT,
play_auto_count INT,
) partitioned by (DataYear STRING, DataMonth STRING, DataDay STRING)
STORED AS ORC
TBLPROPERTIES('orc.create.index'='true','orc.bloom.filter.columns'='media_file_name','orc.bloom.filter.write.version'='or
iginal');

(2) I upgrade the presto server version to 0.229.

Thanks for your help.
